tests/test_routers/admin/test_admin.py

```diff
@@ -231,6 +231,8 @@ def test_update_config(app: FastAPI, wait_for_startup_after):
     initial_contract = initial_config["model_contract"]
     initial_pipelines = initial_config["pipelines"]
     jsonl_file_path = f"{initial_config['artifact_path']}/config_history.jsonl"
+    with jsonlines.open(jsonl_file_path, "r") as reader:
+        initial_config_count = len((list(reader)))
```

Comment:
Oups, extra parentheses:
```suggestion
        initial_config_count = len(list(reader))
```